homeassistant/components/reolink/__init__.py

```diff
@@ -54,6 +54,9 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
         raise ConfigEntryNotReady(
             f"Error while trying to setup {host.api.host}:{host.api.port}: {str(err)}"
         ) from err
+    except Exception:  # pylint: disable=broad-except
+        await host.stop()
```

Comment:
Can `api.unsubscribe` raise any errors? That's called when calling `host.stop`.

Reply:
Yes it potentially could.
I have added a `try except` block around it to catch any errors I am aware could be possible.